src/backend/distributed/utils/colocation_utils.c

```diff
@@ -939,6 +958,29 @@ ColocatedShardIdInRelation(Oid relationId, int shardIndex)
 
 
 /*
+ * DeleteColocationGroupIfEmpty function deletes given co-location group if there is no
```

Comment:
Minor -- sugesstion feel free to ignore`DeleteColocationGroupIfEmpty ` -> `DeleteColocationGroupIfNoTablesBelong `